### ISSUE TITLE
Purpose of terms.md

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/dwc#82]
@mdoering, @tucotuco: what is the purpose of this file: https://github.com/tdwg/dwc/blob/dwc_cleanup/terms/terms.md Does it appear on the website?

**Comment by mdoering** [comment]
I have no idea and only saw that yesterday for the first time. Looks like it came from the documentation?

**Comment by tucotuco** [comment]
These were copied from the documentation repository and constitute the
auxiliary documentation on the classes from the Google Code site.  This
remains only as a reference until this information can be captured on the
GBIF Media Wiki. Perhaps Éamonn can assess whether this has been done.

**Comment by tucotuco** [comment]
We can close this issue when the data have been secured on the GBIF Media Wiki, for which Éamonn has taken responsibility.

**Comment by tucotuco** [comment]
From email discussions, I think the consensus is converging on managing all commentary on this Github site, and on producing translations from a simple set of csv files.

JRW:
"I think it might be easiest to have a simple csv file with a column for term name and columns for the translations. We could script combining that with the normative document to produce translated versions. I worry a little about making sure the translation is for the current version. It would be good if the translation was tagged with a term version so we knew unequivocally what it is meant to match. In the translated rdf files (or maybe all in one file) we would include the original normative definitions and comments so that people would always have that as a reference."

Éamonn:
"Hi John,
I agree - we should come up with the simplest possible CSV file making sure the term for translation is versioned.
The attached are some spreadsheets of translations that I used in the past. I exported the Excel files in XML and then transformed them using XSLT, combining all translations into one master RDF file with the original normative definitions.  We could probably test this with Japanese as there seemed to be interest when I asked the Japanese Node manager when he was here last week about value of such translations."

I have included the spreadsheets for reference in the resources folder in the dwc_cleanup branch.
